react-toggle-switch-component/my-app/src/ToggleSwitch.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import ToggleSwitch from './ToggleSwitch.css';
+import cx from 'classnames';
+
+const Switch = ({ rounded = false, isToggled, onToggle }) => {
+  const sliderCX = cx('slider', {
+    rounded: rounded,
+  });
+
+  return (
+    <label className="switch">
+      <input type="checkbox" checked={isToggled} onChange={onToggle} />
+      <span className={sliderCX} />
```

Comment:
The following is more succinct:
`<span className={cx('slider', { rounded })} />`